### ISSUE TITLE
improve organization of the repository

### ISSUE THREAD

**Issue by alwil** [Rbanism/geospatial-data-carpentry-tud#1]
Hi @cforgaci, 

I suggest that  we improve the organization of the files in this master repo, creating a separate folder for session documents.
There are also some obsolete files in the root (I think testing of gitautopush?), that ought to be removed if we are to use this as a template/ master files.

**Comment by cforgaci** [comment]
Good point! Let's keep this issue open and do the cleaning as you suggest after the workshop, once we have merged the changes from there.
